torchtitan/models/qwen3/model/model.py

```diff
@@ -170,6 +173,8 @@ def __init__(self, model_args: Qwen3ModelArgs):
         match self.attn_type:
             case "flex":
                 self.inner_attention = FlexAttentionWrapper()
+            case "varlen":
+                self.inner_attention = VarlenAttentionWrapper()
             case _:
```

Comment:
```suggestion
            case "sdpa":
```
and raise ValueError as fallback